spec/controllers/articles_controller_spec.rb

```diff
@@ -1,22 +1,22 @@
 require 'spec_helper'
 
 RSpec.describe ArticlesController, type: :controller do
-  describe '#index' do
+  describe 'GET #index' do
     describe 'show the list to articles saved' do
     end
   end
 
-  describe '#show' do
+  describe 'GET #show' do
     it 'show the articles' do
     end
   end
 
-  describe '#new'do
+  describe 'GET #new'do
     it 'create new post' do
     end
   end
 
-    describe ' #create' do
+    describe 'GET #create' do
         context 'when invalid an article' do
           it 'return article false ' do
```

Comment:
Check context, our scope needs to be consistence, and the expects necessarily are needed to read as a sentence.

Check this:

```
describe 'GET #create' do
  context 'when invalid an article' do
    it 'return article false' do
```
If you group descriptions of the previous specs, you can read:
`GET #create when invalid an article return article false`. This is not appropriate. Telling than when an article is invalid returns a false article is not correct.

Change this descriptions and add the expectations. Don't upload blank specs.